### ISSUE TITLE
Change device_state_attributes to extra_state_attributes

### ISSUE THREAD

**Issue by pdobrien3** [uvjustin/alarmdotcom#55]
Just a heads up that on 2021.12.0b0, seeing these errors:

Logger: homeassistant.helpers.entity
Source: helpers/entity.py:543
First occurred: December 3, 2021, 9:07:53 PM (2405 occurrences)
Last logged: 7:08:25 AM

Entity alarm_control_panel.alarm_com (<class 'custom_components.alarmdotcom.alarm_control_panel.AlarmDotCom'>) implements device_state_attributes. Please report it to the custom component author.

This should provide the fix:
https://discord.com/channels/330944238910963714/427516175237382144/916624064301391892
or the actual comment:
device_state_attributes -> extra_state_attributes, is a name change... its really just that

**Comment by pdobrien3** [comment]
I changed:
```
@property
 def device_state_attributes(self):
     """Return the state attributes."""
     return {"sensor_status": self._alarm.sensor_status}
```
to:
```
@property
 def extra_state_attributes(self):
     """Return the state attributes."""
     return {"sensor_status": self._alarm.sensor_status}
```
and it silenced the error.  Still testing if it broke anything else.

**Comment by uvjustin** [comment]
Fixed in v0.1.11
